packages/neuron-ui/src/components/ConfirmingWrap/index.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+
+import { Confirming } from 'widgets/Icons/icon'
+
+import styles from './ConfirmingWrap.module.scss'
+
+const ConfirmingWrap = ({
+  status,
+  statusText,
+  confirmations,
+}: {
+  status: string
+  statusText: string
+  confirmations?: string
```

Comment:
It uses to show transaction status, how about using `confirmationCount` to replace `confirmations`?